config/routes.rb

```diff
@@ -20,6 +20,13 @@
           post 'submit'
         end
       end
+
+      resources :events, only: [] do
+        collection do
+          get 'festival' => 'events#index', type: 'Event::Festival'
```

Comment:
since it is a collection, shouldn't that also be `festivals`? Here, I am unsure...

Reply:
I decided to pluralize this as well.